src/Fantomas/CodePrinter.fs

```diff
@@ -2526,6 +2545,8 @@ and genExpr astContext synExpr ctx =
 
     expr ctx
 
+and genExprInherit astContext synExpr ctx node = genExpr astContext synExpr ctx
```

Comment:
Please don't introduce unused code.
A PR here should contain the bare minimum change that solve an issue.